website_search/models/website_search.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# pylint: disable=missing-docstring
+import inspect
+from odoo import api, fields, models
+
+
+class WebsiteSearch(models.TransientModel):
+    '''
```

Comment:
doc strings use double quotes. As this is a new module, why not use black on it and minimize changes needed for future migrations to 13.0 and beyond?